pmm-app/tests/DbaaS/pages/dbaasPage.js

```diff
@@ -153,4 +187,195 @@ module.exports = {
       assert.notStrictEqual(actionClass, null, `User Should not be able to Perform ${actionName} on the DB Cluster`);
     }
   },
+
+  async verifyElementInSection(section) {
+    for (const element in section) {
+      I.waitForElement(section[element], 30);
+      I.seeElement(section[element]);
+      if (element !== 'showPasswordButton') {
+        I.seeElement(this.tabs.dbClusterTab.fields.clusterDetailProperty(section[element]));
+      }
+    }
+  },
+
+  async createClusterBasicOptions(k8sClusterName, dbClusterName, dbType) {
+    const dbaasPage = this;
```

Comment:
is there any reason to use `dbaasPage` instead of `this`

Reply:
readability for me, no other reason, I can move code from scenario to page later without much modifications

Reply:
I just observe that you use both `this` and `dbaasPage` in these methods

Reply:
Done